packages/engine/src/scene/systems/LODSystem.tsx

```diff
@@ -45,8 +50,16 @@ function updateLOD(
   }
 }
 
+let engineState: typeof EngineState._TYPE | null
+function reactor() {
+  useEffect(() => {
+    engineState = getState(EngineState)
```

Comment:
This is unnecessary, you can just put `const engineState = getState(EngineState)` at the start of `execute`